README.md

```diff
@@ -43,7 +43,8 @@ let server = new FastBootAppServer({
   gzip: true, // Optional - Enables gzip compression.
   host: '0.0.0.0', // Optional - Sets the host the server listens on.
   port: 4000, // Optional - Sets the port the server listens on (defaults to the PORT env var or 3000).
-  sandboxGlobals: { GLOBAL_VALUE: MY_GLOBAL } // Optional - Make values available to the Ember app running in the FastBoot server, e.g. "MY_GLOBAL" will be available as "GLOBAL_VALUE"
+  sandboxGlobals: { GLOBAL_VALUE: MY_GLOBAL } // Optional - Make values available to the Ember app running in the FastBoot server, e.g. "MY_GLOBAL" will be available as "GLOBAL_VALUE",
+  chunkedResponse: true // Optional - Opt-in to chunked transfer encoding, transferring the head, body and potential shoeboxes in separate chunks.
```

Comment:
May suggest to add context of when to use this.

Reply:
I added a sentence pointing that out. I think though if this proves to be stable it would probably make sense to enable this by default? I can't really think of any negative effects this might have (of course negative effects might show up once people start using this ;) ).

Reply:
Yup we should make it default eventually :)